overview.md

```diff
@@ -0,0 +1,180 @@
+# STAC Overview
+
+There are three component specifications that together make up the core SpatioTemporal Asset Catalog specification.
+Each can be used alone, but they work best in concert with one another. The [STAC API specification](https://github.com/radiantearth/stac-api-spec) 
+builds on top of that core, but is out of scope for this overview. Items are the core atomic unit, representing a 
+single spatiotemporal asset as GeoJSON so it can be searched. Catalogs are structural elements, used to group Items
+and Collections. Collections *are* catalogs, that add more required metadata and describe a group of related Items.
+For more on the differences see the [section below](#catalogs-vs-collections)
+
+## Item Overview
+
+Fundamental to any SpatioTemporal Asset Catalog, an [Item](item-spec/item-spec.md) represents an atomic collection of inseparable 
+data and metadata. A STAC Item is a [GeoJSON](http://geojson.org/) [Feature](https://tools.ietf.org/html/rfc7946#section-3.2)
+and can be easily read by any modern GIS or geospatial library, and it describes a [SpatioTemporal Asset](#what-is-a-spatiotemporal-asset). 
+The STAC [Item JSON specification](item-spec/item-spec.md) includes additional fields for:
+
+* the time the asset represents;
+* a thumbnail for quick browsing;
+* asset links, links to the described data;
+* relationship links, allowing users to traverse other related STAC Items.
+
+A STAC Item can contain additional fields and JSON structures to communicate more information about the
+asset, so it can be easily searched. STAC provides a core set of 
+[Common Metadata](https://github.com/radiantearth/stac-spec/blob/catalog-spec-evolve/item-spec/common-metadata.md)
+and there is a wider community working on a variety of [STAC Extensions](https://github.com/radiantearth/stac-spec/tree/catalog-spec-evolve/extensions) that provide shared metadata for more specific domains. Both aim to describe data with well known, well
+defined terms to enable consistent publishing and better search. For more recommendations on selecting fields
+for an Item see [this section](https://github.com/radiantearth/stac-spec/blob/catalog-spec-evolve/best-practices.md#field-selection-and-metadata-linking) of the best practices document.
```

Comment:
```suggestion
for an Item see [this section](best-practices.md#field-selection-and-metadata-linking) of the best practices document.
```

Reply:
It seems you just committed one of the three suggestions and I guess that was not intended, to I committed them for you ;-)

Reply:
Weird - must have missed some final button I was supposed to push. Thanks!